docs/source/FlatBuffers.md

```diff
@@ -131,6 +131,8 @@ sections provide a more in-depth usage guide.
     in your own programs.
 -   How to [use the generated Go code](@ref flatbuffers_guide_use_go) in your
     own programs.
+-   How to [use the generated C code](@ref flatbuffers_guide_use_c) in your
+    own programs.
```

Comment:
Maybe call this "How to use FlatBuffers in C", as the current wording equivalent to the other languages would appear to suggest that you can use it with C using code generated from the main compiler.